src/main/kotlin/lotto/domain/LottoNumber.kt

```diff
@@ -1,6 +1,6 @@
 package lotto.domain
 
-data class LottoNumber private constructor(private val value: Int) {
+data class LottoNumber(private val value: Int) {
     companion object {
```

Comment:
and now, i think u can apply the `value class` 😄 
https://github.com/herotechcourse/kotlin-lotto/pull/8#discussion_r2193347343